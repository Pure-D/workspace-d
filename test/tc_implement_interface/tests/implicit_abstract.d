module implicit_abstract;

class Foo : Bar
{
}

class Bar
{
	abstract int prop() @property;
}
