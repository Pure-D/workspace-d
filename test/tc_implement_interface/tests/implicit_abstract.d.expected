implement 40
override int prop() @property